src/js/service/ImportService.js

```diff
@@ -0,0 +1,121 @@
+/* Image and Animation import service supporting the import dialogs,
+ * file dropper service, and headless import API. */
```

Comment:
Nice catch, my bad; during initial exploration I thought this service might be used by the file dropper too, but its import behavior is different enough that I didn't hook it up.  It might make sense in the future though, especially if `ImportService` expands to centralize import logic for .piskel files, palettes, etc.